### PR TITLE
improve UI/UX with CSS props

### PR DESCRIPTION
**Pull request by ZakariaRidouh**

### CODE REVIEW

**Comment by xaiki** [comment]
same comment as for the angular version, please post before/after screenshots of how this affects us.

**Comment by vankasteelj** [comment]
no css in the html file please, you can set that in src/app/styl/views/misc.styl, it'll be on the top of the compiled css file

**Comment by vankasteelj** [comment]
There's probably an issue in your commits, as there's no app.css file present

**Comment by xaiki** [comment]
screenshots please

File: src/app/styl/views/main_window.styl [hunk 1]
```diff
@@ -1,8 +1,16 @@
 html,
 body,
+    text-rendering: optimizelegibility
```
Comment (vankasteelj) [inline src/app/styl/views/main_window.styl]: this actually won't work. you're removing height/cursor/user-select/bg-color from body

File: src/app/styl/views/main_window.styl [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
+    -webkit-font-smoothing: subpixel-antialiased;
+    color: #FAFAFA
```
Comment (vankasteelj) [inline src/app/styl/views/main_window.styl]: this is hardcoded color. it's probably not a good idea since we work with themes for the colors

**Comment by vankasteelj** [comment]
I'm failing to see any change, is it normal?

**Pull request closed without merging**